remoteworkitem/scheduler.go

```diff
@@ -61,15 +65,33 @@ func (s *Scheduler) ScheduleAllQueries() {
 					_, err = convert(tx, tq.TrackerID, i, tq.TrackerType)
 					return err
 				})
+				lu = i.LastUpdated
+			}
+			if lu != nil {
+				err := s.updateTrackerQueryWithLastUpdated(tq.TrackerQueryID, lu)
+				if err != nil {
+					log.Println("Couldn't update the last updated value", err)
```

Comment:
I think the current message convey that and details will be available in the `err` object.